test/beacon/lifecycle/template_test.exs

```diff
@@ -4,6 +4,18 @@ defmodule Beacon.Lifecycle.TemplateTest do
   use Beacon.Test
   alias Beacon.Lifecycle
 
+  setup do
+    site = default_site()
+
+    # we aren't passing through PageLive normally in these tests so we have to manually
+    # enable the ErrorHandler and set the site in the Process dictionary
+    # (which would normally happen in the LiveView mount)
+    Process.put(:__beacon_site__, site)
+    Process.flag(:error_handler, Beacon.ErrorHandler)
```

Comment:
I'd prefer to not use public Beacon functions in tests, because if the logic of `enable/1` ever changes, it could affect the results of the test in a way that was not intended.

Reply:
okay